.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Go
+
+on: [push, pull_request]
+
+env:
+  VERSION: 1.13.x
+  GO111MODULE: on
+  QUAY_PATH: quay.io/brancz/kube-rbac-proxy
+
+jobs:
+  create-cluster:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Create cluster
```

Comment:
There is an action for it, which handles kind bootstraping. Take a look at https://github.com/prometheus-operator/prometheus-operator/blob/master/.github/workflows/ci.yaml#L96-L107

Reply:
Fixed this to the new version.